src/app/routes/homePage/getInitialData/index.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Agent } from 'https';
+import KyrgyzHomeFixture from '#data/kyrgyz/homePage/index.json';
+import * as fetchPageData from '../../utils/fetchPageData';
+
+import getInitialData from '.';
+
+process.env.BFF_PATH = 'https://mock-bff-path';
+
+const agent = { cert: 'cert', ca: 'ca', key: 'key' };
+const getAgent = jest.fn(() => Promise.resolve(agent as unknown as Agent));
+
+describe('Home Page - BFF Fetching', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should request local fixture data when the app env is "local"', async () => {
+    process.env.SIMORGH_APP_ENV = 'local';
+
+    const fetchDataSpy = jest.spyOn(fetchPageData, 'default');
+    fetchDataSpy.mockImplementation(() =>
+      Promise.resolve({
+        status: 200,
+        json: JSON.stringify(KyrgyzHomeFixture),
+      }),
+    );
+
+    await getInitialData({
+      path: 'http://localhost/kyrgyz/tipohome',
```

Comment:
Should host be removed from the path?
```suggestion
      path: '/kyrgyz/tipohome',
```

Reply:
Removed!